src/infrastructure/apiSSE.ts

```diff
@@ -1,19 +1,19 @@
 import { GenericModuleParams } from "../common/TagoIOModule";
 import regions from "../regions";
 
-const channelsWithID = ["device_inspector", "analysis_console", "ui_dashboard"] as const;
-const channelsWithoutID = ["notification", "analysis_trigger", "ui"] as const;
+const channelsWithID = ["analysis_console", "device_inspector", "device_data", "ui_dashboard"] as const;
+const channelsWithoutID = ["analysis_trigger", "notification", "ui"] as const;
 const channels = [...channelsWithID, ...channelsWithoutID] as const;
 
 type ChannelWithID = (typeof channelsWithID)[number];
 type ChannelWithoutID = (typeof channelsWithoutID)[number];
 
-type OpenSSEWithID = GenericModuleParams & {
```

Comment:
`token`/`region` (from `GenericModuleParams`) have been moved to the second parameter of `openSSEListening`